test/src/runner/testRunWithTags.js

```diff
@@ -98,16 +98,44 @@ describe('testRunWithTags', function() {
       globals: {
         reporter(results) {
           assert.ok('demoTagTest' in results.modules['tags/sample'].completed);
+          assert.strictEqual(Object.keys(results.modules).length, 1);
         }
       },
-      filter: 'tags/*',
-      tag_filter: ['login'],
+      filter: '**/tags/*',
```

Comment:
There's a problem here. The pattern should match the tests under "sampletests/tags"

Reply:
It seems like this is a separate bug, the fact that in matchers/filename.js the filter is not joined with the specified test folder (sampletests). It is only done so for the first folder specified using the `src_folders` setting. I guess the test has been passing erroneously thus far.